File: src/com/digosofter/digowebapp/websocket/ObjWsInterlocutor.java
package com.digosofter.digowebapp.websocket;

import com.digosofter.digowebapp.objeto.ObjMain;

public final class ObjWsInterlocutor extends ObjMain {

  private int _intFncId = -1;

  private String _strJson;

  public int getIntFuncId() {

    return _intFncId;
  }

  public String getStrJson() {

    return _strJson;
  }

  public void setIntFuncId(int intFuncId) {

    _intFncId = intFuncId;
  }

  public void setStrJson(String strJson) {

    _strJson = strJson;
  }

}

File: JDigo/websocket/ObjWsInterlocutor.js
// CONSTANTE
// FIM CONSTANTE

// ATRIBUTOS GLOBAIS
// FIM ATRIBUTOS GLOBAIS

// CLASSE
function ObjWsInterlocutor() {
  // HERANÇA

  Objeto.apply(this);

  // FIM HERANÇA

  // ATRIBUTO

  var _this = this;

  this._intFncId = 0;

  this.getIntFncId = function() {
    return this._intFncId;
  }

  this.setIntFncId = function(intFncId) {
    this._intFncId = intFncId;
  }

  this._strJson = null;

  this.getStrJson = function() {
    return this._strJson;
  }

  this.setStrJson = function(strJson) {
    this._strJson = strJson;
  }

  // FIM ATRIBUTO

  // MÉTODO
  // FIM MÉTODO

  /* Construtor */
  {

    try {

    } catch (e) {
      new Erro("Erro inesperado.", e);
    }
  }

}